app/models/organizations/business_line.rb

```diff
@@ -540,14 +569,27 @@ def bgs_attorneys_join
       "LEFT JOIN bgs_attorneys ON claimants.participant_id = bgs_attorneys.participant_id"
     end
 
+    def pending_request_issues_join
+      "LEFT JOIN pending_request_issues on pending_request_issues.decision_review_id = tasks.appeal_id
+        AND pending_request_issues.decision_review_type = tasks.appeal_type"
+    end
+
+    # :reek:ControlParameter
+    def pending_issue_filter(query_type)
```

Comment:
You don't need to pass in query_type since it's an accessor on the class.

```suggestion
    def pending_issue_filter
```

Reply:
Thats right... good pointing that out